vmdb/app/models/ems_container.rb

```diff
@@ -3,13 +3,24 @@ class EmsContainer < ExtManagementSystem
     EmsKubernetes
   )
 
+  has_many :container_nodes,    :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_groups,   :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_services, :foreign_key => :ems_id, :dependent => :destroy
+
   def self.supported_subclasses
     subclasses
   end
 
-  # Preload any subclasses of this class, so that they will be part of the
-  #   conditions that are generated on queries against this class.
-  EmsContainer::SUBCLASSES.each { |c| require_dependency Rails.root.join("app", "models", "#{c.underscore}.rb").to_s }
```

Comment:
Was this in here twice?  The diff looks really weird.  Perhaps it's just how the `end`s align